lib/index.js

```diff
@@ -276,17 +276,25 @@ module.exports = class SVGSpritemapPlugin {
         // Clean up the spritemap chunk
         compiler.hooks.emit.tapAsync(plugin, (compilation, callback) => {
             const { output: outputOptions } = this.options;
+            const assets = compilation.assets;
+            
 
             if ( outputOptions.chunk.keep ) {
                 return callback();
             }
 
             compilation.chunks.forEach((chunk) => {
+                const sourceMapStr = `${chunk.files[0]}.map`;
+                
                 if ( chunk.name !== outputOptions.chunk.name ) {
                     return;
                 }
 
-                delete compilation.assets[chunk.files[0]];
+                delete assets[chunk.files[0]];
```

Comment:
This should be `compilation.assets` instead of just `assets`, right?

Reply:
No, I defined `assets` on line [279](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/pull/79/files#diff-6d186b954a58d5bb740f73d84fe39073R279) so we didn't have to repeat `.compilation`

Reply:
Let's keep it at `compilation.assets`, it's a little more verbose which I like because we're making changes to the `compilation` object from the surrounding scope.